frontend/src/console/components/Account/displayMessages.ts

```diff
@@ -0,0 +1,56 @@
+const messages = {
+  explanation: {
```

Comment:
It is needed since the contents are different from those in `src/console/components/NoticeBoard/displayMessages.ts`. Almost each component has its own set of displayMessages.